app/frontend/src/submissions/components/SubmissionPreview/SubmissionPreview.vue

```diff
@@ -0,0 +1,485 @@
+      </b-row>
+      <b-row>
+        <b-col cols="12" lg="4"><span class="font-weight-bold">Depth to Bedrock:</span> {{ form.bedrock_depth }} {{ form.bedrock_depth ? 'feet':''}}</b-col>
+        <b-col cols="12" lg="4"><span class="font-weight-bold">Artesian Pressure:</span> {{ form.artesian_pressure }} {{ form.artesian_pressure ? 'feet': ''}}</b-col>
+      </b-row>
+    </fieldset>
+
+    <fieldset class="my-3 detail-section" v-if="sections.decommissionInformation">
+      <legend>Well Decommission Information</legend>
+      <b-row>
+        <b-col cols="12" lg="4"><span class="font-weight-bold">Finished Well Depth:</span> {{ form.finished_well_depth }} {{ form.finished_well_depth ? 'feet':''}}</b-col>
+        <b-col cols="12" lg="4"><span class="font-weight-bold">Sealant Material:</span> {{ form.sealant_material }}</b-col>
+      </b-row>
+      <b-row>
+        <b-col cols="12" lg="4"><span class="font-weight-bold">Reason for Decommission:</span> {{ form.decommission_reason }}</b-col>
+        <b-col cols="12" lg="4"><span class="font-weight-bold">Backfill Material:</span> {{ form.backfill_material }}</b-col>
+      </b-row>
+      <b-row>
+        <b-col cols="12" lg="4"><span class="font-weight-bold">Method of Decommission:</span> {{ form.decommission_method }}</b-col>
+        <b-col cols="12" lg="4"><span class="font-weight-bold">Decommission Details:</span> {{ form.decommission_details }}</b-col>
+      </b-row>
+    </fieldset>
+
+    <fieldset class="my-3 detail-section">
+      <legend>Comments</legend>
+      <p>
+        {{ form.comments ? form.comments : 'No comments submitted' }}
+      </p>
+      <p>
+        <span class="font-weight-bold">Alternative Specs Submitted:</span> {{ form.alternative_specs_submitted }}
+      </p>
+    </fieldset>
+
+  </div>
+</template>
+
+<script>
+import proj4 from 'proj4'
+import { mapGetters } from 'vuex'
+import CoordsMap from '@/submissions/components/SubmissionForm/CoordsMap.vue'
+
+import filterBlankRows from '@/common/filterBlankRows'
+import codeToDescription from '@/common/codeToDescription.js'
+
+export default {
+  name: 'SubmissionPreview',
+  components: {
+    CoordsMap
+  },
+  mixins: [filterBlankRows, codeToDescription],
+  props: [
+    'form',
+    'activity',
+    'sections'
+  ],
+  computed: {
+    wellSubclass () {
+      let subclassCodes = []
+      let subclass = null
+
+      // generate a list of subclasses
+      if (this.form.well_subclass && this.form.well_class) {
+        this.codes.well_classes.forEach((item) => {
+          if (item.wellsubclasscode_set) {
+            subclassCodes = subclassCodes.concat(item.wellsubclasscode_set)
+          }
+        })
+
+        // find the subclass to get its description
+        subclass = subclassCodes.find((code) => {
+          return code.well_subclass_guid === this.form.well_subclass
+        })['description']
+      }
+
+      return subclass
+    },
+    UTM () {
```

Comment:
I think we should move this code into a common area - it's an almost exact copy of the same code in Coords.vue